Apps/phgsgmail/gsgmail.json

```diff
@@ -792,6 +792,400 @@
+                    "data_type": "string",
+                    "example_values": [
+                        "success"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.internet_message_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "<e3d885faf2cc04f261d3874bafe2dc8afc44a230-10004962-100081937@google.com>"
+                    ],
+                    "contains": [
+                        "internet message id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.email",
+                    "data_type": "string",
+                    "example_values": [
+                        "admin@hermancorp.biz"
+                    ],
+                    "contains": [
+                        "email"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.extract_attachments",
+                    "data_type": "numeric",
+                    "example_values": [
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "Total messages returned: 1"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.internalDate",
+                    "data_type": "string",
+                    "example_values": [
+                        "1619925980000"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.subject",
+                    "data_type": "string",
+                    "example_values": [
+                        "Google Workspace: Your invoice is available for hermancorp.biz"
+                    ],
+                    "column_name": "Subject",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.data.*.from",
+                    "data_type": "string",
+                    "example_values": [
+                        "Google Payments <payments-noreply@google.com>"
+                    ],
+                    "contains": [
+                        "email"
+                    ],
+                    "column_name": "From",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.data.*.to",
+                    "data_type": "string",
+                    "example_values": [
+                        "admin@hermancorp.biz"
+                    ],
+                    "contains": [
+                        "email"
+                    ],
+                    "column_name": "To",
+                    "column_order": 0
```

Comment:
Will these column fields work with the custom widget?

Reply:
Fair point, so originally it was a table, but there seems to be a platform bug where rows get duplicated even if there is only one set of data. So, I pivoted to using a custom widget. Let me just remove those so as to not confuse